interface/usergroup/usergroup_admin_add.php

```diff
@@ -12,7 +12,7 @@
 <html>
 <head>
 
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
1. should be below $css_header include
2. should bring in the bootstrap rtl css if pertinent